test/code/scxsystemlib/memory/memoryinstance_test.cpp

```diff
@@ -695,11 +695,26 @@ class MemoryInstance_Test : public CPPUNIT_NS::TestFixture
                 }
 
                 keyValues["TotalMemory"]     = ToSCXUlong(topTokens[2 + toffset]);
+                keyValues["TotalSwap"]       = ToSCXUlong(topTokens[12 + toffset]);
+#if defined(ppc)
+                /*
+                ----------------------------------------------------------
+                ---------- Output from PowerPC Linux systems: ------------
+                ----------------------------------------------------------
+                KiB Mem :  1716736 total,   400064 free,   276288 used,  1040384 buff/cache
+                KiB Swap:  2097088 total,  2097088 free,        0 used.  1212416 avail Mem
+                */
+
+                keyValues["AvailableMemory"] = ToSCXUlong(topTokens[4 + toffset]);
+                keyValues["UsedMemory"]      = ToSCXUlong(topTokens[6 + toffset]);
+                keyValues["AvailableSwap"]   = ToSCXUlong(topTokens[14 + toffset]);
+                keyValues["UsedSwap"]        = ToSCXUlong(topTokens[16 + toffset]);
+#else
                 keyValues["AvailableMemory"] = ToSCXUlong(topTokens[6 + toffset]);
                 keyValues["UsedMemory"]      = ToSCXUlong(topTokens[4 + toffset]);
-                keyValues["TotalSwap"]       = ToSCXUlong(topTokens[12 + toffset]);
```

Comment:
It's at line 698. Only the available and swap memory token locations differ in the output.